### PR TITLE
feat: added `blocklyHighlighted` CSS class to highlighted block's root…

### PR DESCRIPTION
**Pull request by Shreshthaaa**
# Details :
Added `blocklyHighlighted` CSS class to highlighted block's root svg.

# Fixes:

#8263

### CODE REVIEW

File: core/renderers/common/path_object.ts [hunk 1]
```diff
@@ -168,14 +168,25 @@ export class PathObject implements IPathObject {
...
    * @param enable True if highlighted.
    */
+
+  addClass(element: SVGElement, className:string) {
+    element.classList.add(className);
+  }
+
+  removeClass(element: SVGElement, className: string) {
+    element.classList.remove(className);
+  }
```
Comment (BeksOmega) [inline core/renderers/common/path_object.ts:178]: Could you remove these and call the existing `setClass` method instead?

**Review by BeksOmega — CHANGES_REQUESTED** [review]
Thanks for your work! Just one fix.

**Comment by BeksOmega** [comment]
Heya @Shreshthaaa Are you still interested in working on this? (and your other open PRs?)

**Comment by Shreshthaaa** [comment]
> Heya @Shreshthaaa Are you still interested in working on this? (and your other open PRs?)

Hi @BeksOmega , thank you for following up. I apologize for the delay. I've been quite busy lately but I am still interested in working on this and my other open PRs. I will address the requested changes in a day or two.

**Comment by Shreshthaaa** [comment]
@BeksOmega  Hey, I have done the said changes. Kindly review.

**Review by BeksOmega — CHANGES_REQUESTED** [review]
Just pulled this down locally to test and I don't think this is working for the `renders/geras/path_object`. Can you refactor that to properly call `super`?

**Comment by Shreshthaaa** [comment]
> Just pulled this down locally to test and I don't think this is working for the `renders/geras/path_object`. Can you refactor that to properly call `super`?

Hey, I made changes in `renderers/geras/path_object` itself. Kindly review.

**Review by BeksOmega — CHANGES_REQUESTED** [review]
Thanks for the changes but that doesn't quite do what I asked! In the geras path object it should call `super` to add the new CSS class. The only new code in the gears path object should be setting the `svgPathLight.style.display` values.

**Comment by Shreshthaaa** [comment]
> Thanks for the changes but that doesn't quite do what I asked! In the geras path object it should call `super` to add the new CSS class. The only new code in the gears path object should be setting the `svgPathLight.style.display` values.

Hey, I have called `super` in the geras path object file to call the parent class method. Kindly check if the code looks correct now.

**Comment by BeksOmega** [comment]
Heya @Shreshthaaa are you still interested in working on this?

**Comment by Shreshthaaa** [comment]
> Heya @Shreshthaaa are you still interested in working on this?

Hey @BeksOmega, please review the update. Hopefully, everything is in line now. Thank you!

**Pull request merged**